test/support/analyzer_verification/suppress_if.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ElixirAnalyzer.Support.AnalyzerVerification.SuppressIf do
+  @moduledoc """
+  This is an analyzer extension module to test the option :suppress_if between features/assert_call/common checks
+  """
```

Comment:
Awesome! You're adding all of the analyzer features that I wanted to have but not really implement them myself 🥰